kubejs/server_scripts/create_factory_logistics/recipes.js

```diff
@@ -28,17 +28,18 @@ function registerCreateFactoryLogisticsRecipes(event) {
 	], {
 		A: '#forge:rods/copper',
 		B: '#forge:springs/copper',
-		C: 'gtceu:ulv_machine_hull',
+		C: 'create:andesite_casing',
```

Comment:
what about copper casing? since that's the traditional create fluid-related casing